data/game.py

```diff
@@ -314,6 +315,105 @@ def current_play_result(self):
+        ):
+            result += "_line"
+        elif result == "field_out" and (
+            "pop"
+            in self._current_data["liveData"]["plays"].get("currentPlay", {}).get("result", {}).get("description", "")
+        ):
+            result += "_pop"
+        elif result == "field_out" and (
+            "scores"
+            in self._current_data["liveData"]["plays"].get("currentPlay", {}).get("result", {}).get("description", "")
+        ):
+            result += "_rbi"
+
+        if result == "field_out_fly" and (
+            "scores"
+            in self._current_data["liveData"]["plays"].get("currentPlay", {}).get("result", {}).get("description", "")
+        ):
+            result += "_rbi"
+        elif result == "field_out_ground" and (
+            "scores"
+            in self._current_data["liveData"]["plays"].get("currentPlay", {}).get("result", {}).get("description", "")
+        ):
+            result += "_rbi"
+        elif result == "field_out_line" and (
+            "scores"
+            in self._current_data["liveData"]["plays"].get("currentPlay", {}).get("result", {}).get("description", "")
+        ):
+            result += "_rbi"
+        elif result == "field_out_pop" and (
+            "scores"
+            in self._current_data["liveData"]["plays"].get("currentPlay", {}).get("result", {}).get("description", "")
+        ):
+            result += "_rbi"
+
+        if result == "sacrifice_fly" and (
+            "scores"
+            in self._current_data["liveData"]["plays"].get("currentPlay", {}).get("result", {}).get("description", "")
+        ):
+            result += "_rbi"
+        elif result == "sacrifice_bunt" and (
+            "scores"
+            in self._current_data["liveData"]["plays"].get("currentPlay", {}).get("result", {}).get("description", "")
+        ):
+            result += "_rbi"
+
+        if result == "single" and (
+            "scores"
+            in self._current_data["liveData"]["plays"].get("currentPlay", {}).get("result", {}).get("description", "")
+        ):
+            result += "_rbi"
+        elif result == "double" and (
+            "scores"
+            in self._current_data["liveData"]["plays"].get("currentPlay", {}).get("result", {}).get("description", "")
+        ):
+            result += "_rbi"
+        elif result == "triple" and (
+            "scores"
+            in self._current_data["liveData"]["plays"].get("currentPlay", {}).get("result", {}).get("description", "")
+        ):
+            result += "_rbi"
+        elif result == "walk" and (
+            "scores"
+            in self._current_data["liveData"]["plays"].get("currentPlay", {}).get("result", {}).get("description", "")
+        ):
+            result += "_rbi"
+        elif result == "hit_by_pitch" and (
+            "scores"
+            in self._current_data["liveData"]["plays"].get("currentPlay", {}).get("result", {}).get("description", "")
+        ):
+            result += "_rbi"
```

Comment:
Similarly, I think making a helper function called "did_score" or something could simplify this